File: Software Technologies/Java Basic Syntax Exercises/src/AdvertisementMessage.java
import java.util.Random;
import java.util.Scanner;

public class AdvertisementMessage {
    public static void main(String[] args) {

        Scanner scanner = new Scanner(System.in);

        int numberOfmessages = Integer.parseInt(scanner.nextLine());

        String[] phrases = {"Excellent product.", "Such a great product.",
                "I always use that product.", "Best product of its category.",
                "Exceptional product.", "I can't live without this product."};

        String[] events = {"Now I feel good.", "I have succeeded with this product.",
                "Makes miracles. I am happy of the results!", "I cannot believe but now I feel awesome.",
                "Try it yourself, I am very satisfied.", "I feel great!"};

        String[] author = {"Diana", "Petya", "Stella", "Elena", "Katya", "Iva", "Annie", "Eva"};
        String[] cities = {"Burgas", "Sofia", "Plovdiv", "Varna", "Ruse"};
        for (int i = 0; i < numberOfmessages; i++) {


            int rndmPhrases = new Random().nextInt(phrases.length);
            int rndmEvent = new Random().nextInt(events.length);
            int rndmAuthor = new Random().nextInt(author.length);
            int rndmCities = new Random().nextInt(cities.length);

            String randomPhrases = phrases[rndmPhrases];
            String randomEvent = events[rndmEvent];
            String randomAuthor = author[rndmAuthor];
            String randomCities = cities[rndmCities];


            System.out.printf("%S %S %S %S", randomPhrases, randomEvent, randomAuthor, randomCities);
            System.out.println();
        }

    }
}

File: Programming Fundamentals/Programming-Fundamentals-September-2017/ObjectsАndClasses-Exercise/02.AdvertisementMessage/AdvertisementMessage.cs
namespace _02.AdvertisementMessage
{
    using System;
    using System.Collections.Generic;
    public class AdvertisementMessage
    {
        public static void Main()
        {
            List<string> phrases = new List<string>()
            {
                "Excellent product.",
                "Such a great product.",
                "I always use that product.",
                "Best product of its category.",
                "Exceptional product.",
                "I can’t live without this product." 
            };
            List<string> events = new List<string>()
            {
                "Now I feel good.",
                "I have succeeded with this product.",
                "Makes miracles. I am happy of the results!",
                "I cannot believe but now I feel awesome.",
                "Try it yourself, I am very satisfied.",
                "I feel great!" 
            };
            List<string> authors = new List<string>()
            {
               "Diana",
                "Petya",
                "Stella",
                "Elena",
                "Katya",
                "Iva",
                "Annie",
                "Eva"
            };
            List<string> cities = new List<string>()
            {
                "Burgas",
                "Sofia",
                "Plovdiv",
                "Varna",
                "Ruse"
            };
            Random rnd = new Random();
            int numberPhrases = int.Parse(Console.ReadLine());

            for (int i = 0; i < numberPhrases; i++)
            {
                int eve = rnd.Next(events.Count);
                string temp1 = events[eve];
                int auth = rnd.Next(authors.Count);
                string temp2 = authors[auth];
                int citi = rnd.Next(cities.Count);
                string temp3 = cities[citi];
                int prh = rnd.Next(phrases.Count);
                string temp4 = phrases[prh];

                Console.WriteLine("{0}{1}{2} - {3}",temp4, temp1, temp2, temp3);
            }
        }
    }
}
